packages/h5py/patches/0001-Fix-incompatible-pointer-type.patch

```diff
@@ -0,0 +1,57 @@
+From 30ba08deaa551b0920530f6b2edac59f2dd9e28e Mon Sep 17 00:00:00 2001
```

Comment:
Did you upstream this?

Reply:
Yes, I did (not merged yet): https://github.com/h5py/h5py/pull/2142